File: ProgrammingFundamentals/Java/02.ConditionalStatementsAndLoops/Exercises/src/P13_GameOfNumbers.java
import java.util.Scanner;
public class P13_GameOfNumbers {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);
        int n = Integer.parseInt(console.nextLine());
        int m = Integer.parseInt(console.nextLine());
        int magicNum = Integer.parseInt(console.nextLine());
        int sum = 0, count = 0;

        for(int i = n; i <= m; i++) {
            for(int j = n; j <= m; j++) {
                count++;
                sum = i + j;
                if(sum == magicNum) {
                    System.out.format("Number found! %d + %d = %d", j, i, magicNum);
                    return;
                }
            }
        }
        System.out.format("%d combinations - neither equals %d", count, magicNum);
    }
}

File: ProgrammingFundamentals/C#/02.ConditionalStatementsAndLoops/Exercises/P13_GameOfNumbers/P13_GameOfNumbers.cs
using System;

namespace P13_GameOfNumbers
{
    class P13_GameOfNumbers
    {
        static void Main(string[] args)
        {
            int n = int.Parse(Console.ReadLine());
            int m = int.Parse(Console.ReadLine());
            int magicNum = int.Parse(Console.ReadLine());
            int sum = 0, count = 0;

            for (int i = n; i <= m; i++)
            {
                for (int j = n; j <= m; j++)
                {
                    count++;
                    sum = i + j;
                    if (sum == magicNum)
                    {
                        Console.WriteLine($"Number found! {j} + {i} = {magicNum}");
                        return;
                    }
                }
            }
            Console.WriteLine($"{count} combinations - neither equals {magicNum}");
        }
    }
}
